packages/@react-aria/grid/src/index.ts

```diff
@@ -23,3 +18,12 @@ export {useGridCell} from './useGridCell';
 export {useGridSelectionCheckbox} from './useGridSelectionCheckbox';
 export {useHighlightSelectionDescription} from './useHighlightSelectionDescription';
 export {useGridSelectionAnnouncement} from './useGridSelectionAnnouncement';
+
+export type {GridProps, GridAria} from './useGrid';
+export type {GridCellAria, GridCellProps} from './useGridCell';
+export type {GridRowGroupAria} from './useGridRowGroup';
+export type {GridRowProps, GridRowAria} from './useGridRow';
```

Comment:
I noticed that the aria component hook options now follow "AriaCOMPONENTOptions" or "AriaCOMPONENTProps". At the risk of being overly pedantic, should the grid hook props follow this same convention?

Just trying to clarify the naming convention for the future

Reply:
Yes ideally, but, I couldn't change some of them that were already exported...

I think the convention is:

* Aria___Props if it's props. Most of these come from `@react-types`
* Aria___Options if we extended the props to add additional options only for the hook (e.g. additional refs)
* ___Aria for the return types

Reply:
Ah right, GridProps were already exported, I'm fine keeping the other grid props naming in line with that then. Thanks for the clarification